vector/v.random/testsuite/test_v_random.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class TestVRandom(TestCase):
+    output = "test01"
+    output2 = "test02"
+    npoints = 100
+    state = "nc_state"
+    zmin = 10
+    zmax = 120
+
+    @classmethod
+    def setUpClass(cls):
+        cls.use_temp_region()
+        cls.runModule("g.region", vector=cls.state)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.del_temp_region()
+
+    def tearDown(cls):
+        cls.runModule("g.remove", type="vector", flags="f", name=cls.output)
+        cls.runModule("g.remove", type="vector", flags="f", name=cls.output2)
+
+    # Checking if number of points equals 100

```

Comment:
Make this a docstring as in other test files. Then the/a framework can pick it up and show it in the test results. Same for the other functions.

The text of my comment is copy pasted from https://github.com/OSGeo/grass/pull/1464#discussion_r596479957 and https://github.com/OSGeo/grass/pull/704#discussion_r438477167.